core/app/models/spree/payment_method/bogus_credit_card.rb

```diff
@@ -57,13 +57,12 @@ def capture(_money, authorization, _gateway_options)
       end
     end
 
-    def void(_response_code, _credit_card, _options = {})
-      ActiveMerchant::Billing::Response.new(true, SUCCESS_MESSAGE, {}, test: true, authorization: AUTHORIZATION_CODE)
-    end
-
-    # @see Spree::PaymentMethod#try_void
-    def try_void(_payment)
-      ActiveMerchant::Billing::Response.new(true, SUCCESS_MESSAGE, {}, test: true, authorization: AUTHORIZATION_CODE)
+    def void(_response_code, _credit_card, options = {})
```

Comment:
Should we add/update tests for that in the bogus credit card specs?

Reply:
At the moment, there are no specs for this class: https://github.com/solidusio/solidus/blob/c6050d71707063d1e46693f7d388ebc4c07d1a60/core/spec/models/spree/payment_method/bogus_credit_card_spec.rb

Reply:
This is tested now, but in the payment method specs, see the second commit. We could move these tests into the bogus own spec files, but I tend to think it has more value if we leave them together in the payment_method specs. Still open to changing my mind, though.